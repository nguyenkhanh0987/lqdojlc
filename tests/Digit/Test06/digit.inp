100
32116 45245
47032 95272
67452 95665
10186 46880
52594 57370
16317 72323
26666 94753
71568 81721
49282 76787
52457 83242
50552 85998
94322 99856
45112 47973
5768 67442
7862 91913
73262 99759
91852 99481
98945 99372
16612 28525
26493 58823
53281 94578
25081 62936
4451 67341
57844 95581
43291 64915
1743 86020
78950 79733
92281 93863
19624 53086
43000 64767
20968 25025
85303 86498
94476 95966
1863 74432
52600 69175
3224 43876
92113 98609
63134 80968
58705 69844
39297 80328
70360 84268
9748 29449
44770 57332
88087 91825
42172 50306
67123 70377
50272 80386
1145 39714
48366 59603
68625 84901
22803 37606
19295 26804
45213 76960
11565 94831
5103 78650
57846 89641
98334 99973
41390 68248
59774 64849
68344 69453
9247 22780
22795 52223
50879 68843
78645 95873
8647 10063
41000 94212
4053 62872
81606 86304
90295 96336
5957 57348
90639 94274
8446 34606
47325 75587
52353 70063
88638 99821
19296 59766
41532 72244
21066 66318
71668 93436
81919 97579
94892 99130
83024 83388
86277 90976
92514 93064
13411 34160
91833 93612
17149 63368
44471 59732
12148 19209
80303 92822
87770 95431
96758 99215
45821 50929
85469 91817
5856 9095
97825 98077
4165 65577
32171 67788
85854 88197
82596 94187
