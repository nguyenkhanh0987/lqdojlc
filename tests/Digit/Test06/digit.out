1585
3093
1246
5458
392
6806
5828
501
1879
2037
2127
235
404
8634
9267
1087
236
62
2230
4073
2369
4625
8912
1973
1988
10303
135
4
4339
2030
410
46
17
9794
1396
6972
162
1146
902
3367
531
3547
1332
178
1072
407
2069
7071
1190
731
2299
1036
2578
8441
9512
1933
102
2399
120
181
2390
4033
1381
545
444
3561
8744
92
55
7832
13
4438
2214
1522
365
5260
2667
5237
866
461
174
9
273
0
3506
5
5975
1773
1522
362
228
147
790
303
964
11
8833
3665
147
367
